tinygrad/tensor.py

```diff
@@ -435,6 +461,7 @@ def dot(self, w:Tensor) -> Tensor:
 
   def contiguous(self): return mlops.Contiguous.apply(self)
   def log(self): return mlops.Log.apply(self)
+  def log2(self): return mlops.Log.apply(self)/0.69314718056
```

Comment:
* (math.log(math.e)/math.log(2)) for readability.

Reply:
@geohot 0.69314718056 is math.log(2), to change base from math.e i divided it by log of 2